include/maxscale/config.hh

```diff
@@ -137,6 +137,7 @@ public:
     std::string   admin_pam_rw_service;         /**< PAM service for read-write users */
     std::string   admin_pam_ro_service;         /**< PAM service for read-only users */
     std::string   admin_ssl_key;                /**< Admin SSL key */
+    std::string   admin_ssl_cipher;             /**< Admin allowed SSL ciphers */
```

Comment:
Since the original issue is not with TLS cipher configuration and the code behaves as if this was similar to the `ssl_version` parameter of listeners and servers, this should be named `admin_ssl_version` and changed into `mxb::ssl_version::Version`.

Having one variable do both is convenient from the implementation point of view but it's inconsistent from the user's point of view. If one types the name of the TLS version wrong, you'll most likely get a less than obvious error about the REST API not being able to start up.